### PR TITLE
Fixing the issue pods try to get scheduled to master nodes

### PR DESCRIPTION
**Pull request by weinliu**
@jhou1 @sunilcio @lyman9966 
Would you mind taking a look?

Execution log with the fix: http://pastebin.test.redhat.com/828686

### CODE REVIEW

**Comment by weinliu** [comment]
/assign @pruan-rht , similar issued as fixed recently, could you help?
Thanks.

**Comment by pruan-rht** [comment]
@weinliu LGTM, but can you please ask others from your team to review it as well before I merge it?

**Comment by jhou1** [comment]
On the bottom of the log http://pastebin.test.redhat.com/828686, it says the step is undefined.

```
1 scenario ([33m1 undefined[0m)
18 steps ([36m13 skipped[0m, [33m1 undefined[0m, [32m4 passed[0m)
0m34.122s
[33m[0m
[33mYou can implement step definitions for undefined steps with these snippets:[0m
[33m[0m
[33mGiven(/^environment has at least (\d+) schedulable worker nodes$/) do |arg1|[0m
[33m  pending # Write code here that turns the phrase above into concrete actions[0m
[33mend[0m
```

File: features/admin/pod.feature [hunk 1]
```diff
@@ -218,7 +218,7 @@ Feature: pod related features
       | p | {"metadata":{"annotations": {"openshift.io/node-selector": ""}}}|
     Then the step should succeed
     Given I store the schedulable nodes in the :nodes clipboard
-    Given environment has at least 2 schedulable nodes
+    Given environment has at least 2 schedulable worker nodes
```
Comment (jhou1) [inline features/admin/pod.feature]: You can add your name to the author, for example `# @author weinliu@redhat.com` to count as a contributor.

Comment (jhou1) [reply features/admin/pod.feature]: After this line https://github.com/openshift/verification-tests/blob/6bec9295c6fdb68e15432961490eba6ad06c671e/features/admin/pod.feature#L210

**Comment by weinliu** [comment]
> On the bottom of the log http://pastebin.test.redhat.com/828686, it says the step is undefined.
> 
> ```
> 1 scenario (�[33m1 undefined�[0m)
> 18 steps (�[36m13 skipped�[0m, �[33m1 undefined�[0m, �[32m4 passed�[0m)
> 0m34.122s
> �[33m�[0m
> �[33mYou can implement step definitions for undefined steps with these snippets:�[0m
> �[33m�[0m
> �[33mGiven(/^environment has at least (\d+) schedulable worker nodes$/) do |arg1|�[0m
> �[33m  pending # Write code here that turns the phrase above into concrete actions�[0m
> �[33mend�[0m
> ```

Updated, now there's no `undefined` issue. log:  http://pastebin.test.redhat.com/829467

**Pull request merged**